### PR TITLE
Add french translations

### PR DESCRIPTION
**Pull request by Antoine-Lalechere**
@MonikaFu 

PR ready to review :)

### CODE REVIEW

File: inst/extdata/PA2024CH_fr_exec_summary/scorecard.Rmd [hunk 1]
```diff
@@ -184,7 +182,7 @@ Ce score représente l'alignement agrégé estimé des secteurs PACTA (hors cime
 
 \subsection{Engagements vérifiés en faveur du net zéro}
 
-Companies are increasingly committing voluntarily to transitioning to net zero and setting interim targets. The effectiveness of such commitments depends on whether interim emissions reduction targets applied are credible, science-based, transparent, and supported by credible action to cut emissions. The following information is based on information from SBTi \textbf{\href{https://sciencebasedtargets.org/companies-taking-action}{(link)}}.
+Companies are increasingly making voluntary commitments to transition to net-zero and set interim targets. The effectiveness of such commitments depends on whether interim targets are credible, science-based, and transparent. The following information is based on information from SBTi \textbf{\href{https://sciencebasedtargets.org/companies-taking-action}{(link)}}.
```
Comment (MonikaFu) [inline inst/extdata/PA2024CH_fr_exec_summary/scorecard.Rmd:185]: This part should be translated to French.

**Review by MonikaFu — CHANGES_REQUESTED** [review]

**Comment by MonikaFu** [comment]
@Antoine-Lalechere see my comment.

File: inst/extdata/PA2024CH_fr_exec_summary/scorecard.Rmd [hunk 2]
```diff
@@ -304,7 +301,7 @@ if (data_net_zero_available) {
 ```{=latex}
 \subsection{Credible Climate Stewardship}
```
Comment (MonikaFu) [inline inst/extdata/PA2024CH_fr_exec_summary/scorecard.Rmd:302]: This should be translated as well

**Review by MonikaFu — CHANGES_REQUESTED** [review]

File: inst/extdata/PA2024CH_fr_exec_summary/template.Rmd [hunk 3]
```diff
@@ -649,6 +648,9 @@ La mesure dans laquelle vous utilisez vos résultats dépend de vous. Cependant,
 \item Des détails concernant les modalités d'utilisation des résultats pour les rapports réalisés selon le TCFD, le SFDR de l'UE ou la taxonomie de l'UE sont disponibles dans le rapport "The Disclosure Puzzle – the role of PACTA" \textbf{(\href{https://2degrees-investing.org/wp-content/uploads/2020/03/PACTA-disclosures-report.pdf}{lien})}.
 \end{itemize}
```
[the remark this answers is not in the document]
Comment (Antoine-Lalechere) [reply inst/extdata/PA2024CH_fr_exec_summary/template.Rmd]: changes done @MonikaFu

File: inst/extdata/PA2024CH_fr_exec_summary/template.Rmd [hunk 4]
```diff
@@ -96,9 +95,9 @@ number_peers <- readr::read_csv(
...
... progrès réalisés par la place financière suisse et les différentes institutions financières participantes en vue d’atteindre les objectifs climatiques. \startunderscoreletter `r gsub("&", "\\&", stringr::str_to_title(investor_name), fixed = TRUE)` \stopunderscoreletter a été invité à participer au test par l'OFEV \textbf{(\href{https://www.bafu.admin.ch/bafu/fr/home/themes/climat/info-specialistes/climat-et-marche-financier/pacta.html}{lien})} et le SIF \textbf{(\href{https://www.sif.admin.ch/sif/fr/home/finanzmarktpolitik/finance-durable.html}{lien})} avec le soutien des associations, en vue d’un examen volontaire de ses portefeuilles financiers. Le test PACTA associe une évaluation quantitative des portefeuilles mondiaux d'actions et d'obligations d'entreprises cotées ainsi que des portefeuilles immobiliers et hypothécaires suisses à une évaluation qualitative des actions climatiques. Au total, 133 institutions financières ont participé au test, dont \startunderscoreletter `r number_peers` \stopunderscoreletter étaient des pairs (c'est-à-dire des participants du même secteur financier que les institutions de prévoyance, les assurances, les banques et les gestionnaires d'actifs).
```
Comment (MonikaFu) [inline inst/extdata/PA2024CH_fr_exec_summary/template.Rmd:98]: Is this translation based on the new English version? It seems to me that it uses old number of participating institutions (it should be 122) and it misses som `textbf{}` marks from the current version. The intro changed quite a lot this time so it should all be translated again I believe. See the English version between tags `UPDATED 1 and UPDATED 1 END`.

**Review by MonikaFu — CHANGES_REQUESTED** [review]
It seems to me from a brief look that a lot of parts between `UPDATED` tags from [English version of the ES](https://github.com/RMI-PACTA/pacta.executive.summary/blob/main/inst/extdata/PA2024CH_en_exec_summary/template.Rmd) in `template.Rmd` is missing. Can you please check again if you covered all of them?

File: inst/extdata/PA2024CH_fr_exec_summary/template.Rmd [hunk 5]
```diff
@@ -95,10 +95,11 @@ number_peers <- readr::read_csv(
...
...50} afin de remplir ses engagements en vertu de la loi suisse sur le climat et l'innovation \textbf{\ref{https://www.fedlex.admin.ch/eli/fga/2022/2403/fr}{(link)}}. Le test climatique 2024 de la PACTA fournit des informations cohérentes et comparables sur le marché financier suisse et sur les progrès réalisés par les différentes institutions financières participantes pour atteindre ces objectifs climatiques. \startunderscoreletter `r gsub("&", "\\&", stringr::str_to_title(investor_name), fixed = TRUE)` \stopunderscoreletter - \startunderscoreletter `r gsub("&", "\\&", stringr::str_to_title(peer_group), fixed = TRUE)` \stopunderscoreletter a été invitée au test par l'OFEV \textbf{\href{https://www. bafu.admin.ch/bafu/fr/home/topics/climate/info-specialists/climate-and-financial-markets/pacta.html}{(link)}} et le SIF, avec le soutien des associations, à faire examiner volontairement ses portefeuilles financiers. Le test PACTA combine une évaluation quantitative des portefeuilles d'actions et d'obligations d'entreprises cotées en bourse, ainsi que des portefeuilles immobiliers et hypothécaires suisses, avec une enquête qualitative sur les stratégies et les actions en faveur du climat.
```
Comment (MonikaFu) [inline inst/extdata/PA2024CH_fr_exec_summary/template.Rmd:98]: change `\ref` to '\href`

File: inst/extdata/PA2024CH_fr_exec_summary/template.Rmd
Comment (MonikaFu) [inline inst/extdata/PA2024CH_fr_exec_summary/template.Rmd:1]: Please remove, 'TO BE REVIEWED' line.

Comment (MonikaFu) [inline inst/extdata/PA2024CH_fr_exec_summary/template.Rmd:1]: Line 390 - there are still references to GECO 2021

Comment (MonikaFu) [inline inst/extdata/PA2024CH_fr_exec_summary/template.Rmd:1]: See `UPDATED 4:` - the footnote seems not to be updated correctly as it mentiond GECO 2021 as well in the English version and I don't see it in the French.

**Review by MonikaFu — CHANGES_REQUESTED** [review]

**Comment by MonikaFu** [comment]
@Antoine-Lalechere please see my comments

**Review by MonikaFu — APPROVED** [review]
LGTM. However I am not able to check the translation language-wise. The summary does compile and I don't see obvious references to the previous COP.

**Pull request merged**